app/controllers/v0/letters_controller.rb

```diff
@@ -8,11 +10,16 @@ def index
              meta: response.metadata
     end
 
-    # :nocov:
     def show
-      head :ok
+      unless EVSS::Letters::Letter::LETTER_TYPES.include? params[:id]
+        raise Common::Exceptions::RecordNotFound, params[:id]
```

Comment:
EVSS returns 404 for this but I agree argument error makes more sense as all the types are known.